tests/foreman/api/test_errata.py

```diff
@@ -17,16 +17,159 @@
 
 # For ease of use hc refers to host-collection throughout this document
 
-from robottelo.decorators import stubbed, tier3
+from nailgun import entities
+from robottelo.cli.factory import (
+    setup_org_for_a_custom_repo,
+    setup_org_for_a_rh_repo,
+)
+from robottelo.constants import (
+    DEFAULT_ARCHITECTURE,
+    DEFAULT_RELEASE_VERSION,
+    FAKE_1_CUSTOM_PACKAGE,
+    FAKE_2_CUSTOM_PACKAGE,
+    FAKE_2_ERRATA_ID,
+    FAKE_3_ERRATA_ID,
+    FAKE_3_YUM_REPO,
+    FAKE_6_YUM_REPO,
+    PRDS,
+    REAL_0_ERRATA_ID,
+    REAL_0_RH_PACKAGE,
+    REAL_1_ERRATA_ID,
+    REAL_2_ERRATA_ID,
+    REPOS,
+    REPOSET,
+)
+from robottelo.decorators import (
+    bz_bug_is_open,
+    run_in_one_thread,
+    skip_if_not_set,
+    stubbed,
+    tier3,
+)
 from robottelo.test import APITestCase
+from robottelo.api.utils import enable_rhrepo_and_fetchid, promote
+from robottelo.vm import VirtualMachine
+from time import sleep
 
+CUSTOM_REPO_URL = FAKE_6_YUM_REPO
+CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
 
+
+@run_in_one_thread
 class ErrataTestCase(APITestCase):
     """API Tests for the errata management feature"""
 
-    @stubbed()
+    @classmethod
+    @skip_if_not_set('clients', 'fake_manifest')
+    def setUpClass(cls):
+        """Create Org, Lifecycle Environment, Content View, Activation key"""
+        super(ErrataTestCase, cls).setUpClass()
+        cls.org = entities.Organization().create()
+        cls.env = entities.LifecycleEnvironment(
+            organization=cls.org).create()
+        cls.content_view = entities.ContentView(
+            organization=cls.org).create()
+        cls.activation_key = entities.ActivationKey(
+            environment=cls.env,
+            organization=cls.org,
+        ).create()
+        setup_org_for_a_rh_repo({
+            'product': PRDS['rhel'],
+            'repository-set': REPOSET['rhst7'],
+            'repository': REPOS['rhst7']['name'],
+            'organization-id': cls.org.id,
+            'content-view-id': cls.content_view.id,
+            'lifecycle-environment-id': cls.env.id,
+            'activationkey-id': cls.activation_key.id,
+        })
+        cls.custom_entities = setup_org_for_a_custom_repo({
+            'url': FAKE_6_YUM_REPO,
+            'organization-id': cls.org.id,
+            'content-view-id': cls.content_view.id,
+            'lifecycle-environment-id': cls.env.id,
+            'activationkey-id': cls.activation_key.id,
+        })
+
+    def _install_package(self, clients, host_ids, package_name):
+        """Workaround BZ1374669 and install package via CLI while the bug is
+        open.
```

Comment:
It's helper to just install a package (to be able to populate errata for it later, right). The thing is these are api tests so it would be great to install package via api, but there's a bug blocking this functionality. So while the bug is not closed - installing package via cli, as soon as the bug is closed tests will use api for package installation. That's the idea